apps/prairielearn/src/sync/fromDisk/courseInstances.ts

```diff
@@ -30,7 +31,6 @@ function getParamsForCourseInstance(courseInstance: CourseInstance | null | unde
   return {
     uuid: courseInstance.uuid,
     long_name: courseInstance.longName,
-    number: courseInstance.number,
```

Comment:
Huh, this was a bug in the previous `CourseInstance` type?

Reply:
As far as I am aware, the previous `CourseInstance` type did not have a number.

Reply:
Looks like it actually did, in error though:

https://github.com/PrairieLearn/PrairieLearn/blob/479137a713237335d851aa9909a746deef6a2e23/apps/prairielearn/src/sync/course-db.ts#L250

Reply:
I see. I meant on the original JSON Schema.

Reply:
Unresolving for now, I'd like to shift some of this cleanup into a separate PR that we land ahead of this one.

Reply:
These were fixed in #11789.